tools/accuracy_checker/configs/bert-large-uncased-whole-word-masking-squad-int8-onnx-0001.yml

```diff
@@ -0,0 +1,45 @@
+models:
+  - name: bert-large-uncased-whole-word-masking-squad-int8-onnx-0001
+    launchers:
+      - framework: dlsdk
+        adapter:
+          type: bert_question_answering
+          start_token_logits_output: "5211"
+          end_token_logits_output: "5212"
+        inputs:
+          - name: "result.1"
+            type: INPUT
+            value: "input_ids"
+          - name: "result.2"
+            type: INPUT
+            value: 'input_mask'
+          - name: "result.3"
+            type: INPUT
+            value: 'segment_ids'
+
+    datasets:
+      - name: squad
+        data_source: squad_v1.1
+        annotation_conversion:
+          converter: squad
+          testing_file: squad_v1.1/dev-v1.1.json
+          vocab_file: bert-large-uncased-whole-word-masking-squad-onnx/vocab.txt
+          max_seq_length: 384
+          max_query_length: 64
+          doc_stride: 128
+          lower_case: true
```

Comment:
please move dataset description to dataset_definition.yml and remove from configs (here only name field should be stayed, `data_source` and `annotation_conversion` should be placed in definitions
BTW, data_source is not used here, we read data from converted annotation, no need in this field